### PR TITLE
several easy homogeneous results

### PR DESCRIPTION
**Pull request by StevenClontz**
Addresses several things in #787

### CODE REVIEW

**Comment by prabau** [comment]
The justification for S86 was incorrect (wrong intervals).  Added an easier one.

**Review by prabau — APPROVED** [review]

**Comment by prabau** [comment]
@StevenClontz For some reason I can't even do a Reset right now in the Advanced tab of the pi-base web page.  It should be unrelated to this PR.  Do you see the same problem?

File: spaces/S000044/properties/P000086.md [hunk 1]
```diff
@@ -4,5 +4,4 @@ property: P000086
 value: false
 ---
 
-$1/3$ belongs to a single open set $(0,1)$, but
-$2/5$ belongs to two open sets $(0,1)$ and $(0,1/2)$.
+$1$ is a closed point, but $1/3$ is not.
```
Comment (StevenClontz) [inline spaces/S000044/properties/P000086.md:7]: $1$ doesn't belong to the space, so this needs to be fixed. I'll address in #791

**Comment by prabau** [comment]
Good catch!

**Pull request merged**